repository/Seaside-WebSocket-Core.package/WAWebSocketPusher.class/instance/socketsDo..st

```diff
@@ -0,0 +1,4 @@
+protected
+socketsDo: aOneArgumentBlock
+	self mutex critical: [
```

Comment:
Would a `^` be helpful here so the sender can easily access the return value from the provided block?